### ISSUE TITLE
False alarm on "for free" cliche

### ISSUE THREAD

**Issue by suchow** [amperser/proselint#1190]
```
			\item Combine the equations to write the EOM in standard form with the forcing component on the right-hand side. For free vibration, the forcing component is 0. 
```

https://github.com/ARTS-Laboratory/Open_Vibrations/blob/8ed33663e05c2d2679cd08dd534f6beba71bfc44/1_Fundamentals_of_Vibrations/Chapter_1_fundamentals_of_vibrations.tex#L335

- For free speech, the issue is more complex.
- For free cheese, call 1 (800) CHEEZEY
- For free men will always be…
